moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1164,6 +1222,107 @@ bool MoveItConfigData::inputKinematicsYAML(const std::string& file_path)
   return true;  // file created successfully
 }
 
+// ******************************************************************************************
+// Helper function for parsing ros_controllers.yaml file
```

Comment:
be more descriptive in comment, currently this comment is identical to line 1272. what is different about this function from processROSControllers?